deployment/terraform/storage.tf

```diff
@@ -17,7 +17,29 @@ resource "aws_s3_bucket" "static" {
 
 resource "aws_s3_bucket" "storage" {
   bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
-  acl    = "private"
+  acl    = "public-read"
+  policy = <<POLICY
```

Comment:
Cool.  But I'm not quite getting how to then attach the policy to the resource in a way that supplies the `var.bucket_name` value.

Taking out the `resources` clause from the policy_document definition then doing 
```
resource "aws_s3_bucket_policy" "storage_anonymous_read" {
  bucket = "${aws_s3_bucket.storage.id}"
  policy = "${data.aws_iam_policy_document.anonymous_read_bucket_policy.json}"
}
```
seems like it might work (terraform successfully builds a plan and the changes look reasonable), but I don't know if that's correct and in good order.